SCAutolib/cli_commands.py

```diff
@@ -165,7 +201,102 @@ def cleanup(ctx):
     exit(ReturnCode.SUCCESS.value)
 
 
-cli.add_command(setup_ca)
-cli.add_command(prepare)
-cli.add_command(setup_user)
-cli.add_command(cleanup)
+@cli.group(cls=NaturalOrderGroup, chain=True)
+@click.pass_context
+def gui(ctx):
+    """ Run GUI Test commands """
+    pass
+
+
+@gui.command()
+def init():
+    """ Initialize GUI for testing """
+    return "init"
+
+
+@gui.command()
+@click.argument("name")
+def assert_text(name):
+    """ Check if a word is found on the screen """
+    return f"assert_text:{name}"
+
+
+@gui.command()
+@click.argument("name")
+def assert_no_text(name):
```

Comment:
I guess we can combine them and use --no flag or something like that. Although that way is okay as well.

Reply:
I started to add a "--no/not" option but, sided with just mirroring the GUI methods.   If you prefer, I can go back to the "--no" option for the assert_text call.